README.md

```diff
@@ -0,0 +1,59 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo List app
+
+> A simple todo list web app build with Javascript.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- PC with a browser and Internet
+- [Node.js](https://nodejs.org/en/download) installed
+### Setup
+
+### Install
+npm install
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
```

Comment:
- [ ] Please remove these headers as there are no content available for them. This is for keeping your README clean and professional 😉